components/GameForm.jsx

```diff
@@ -52,7 +67,23 @@ const StyledForm = styled.form`
 
 const StyledButtonRow = styled.div`
   display: flex;
+  a {
+    flex: 1;
+    text-decoration: none;
+    text-align: center;
+    border: solid 0.1rem grey;
+    border-radius: 5px;
+    &:hover {
+      background-color: lightgrey;
+    }
+  }
   button {
     flex: 1;
+    border: solid 0.1rem grey;
+    border-radius: 5px;
+    background-color: white;
+    &:hover {
+      background-color: lightgrey;
+    }
```

Comment:
The styles you apply here to the anchor and button tag seem to be the exact same as in `pages/[gameId]/index.jsx`.
Moreover, having nested styles makes the wrapping element, `StyledButtonRow` in this case, less flexible and less reusable.

It's not ideal to write code twice, especially if there is an easy solution of having a styled component for your anchor tag and one for your button tag that you only need to write once and can reuse several times in your application.

If you are unsure about how to style an existing component like the `Link` component provided by Next.js, the syntax is:
`const NameOfTheComponent = styled(Link)`